migrations/versions/e6f315c1f2f6_.py

```diff
@@ -0,0 +1,30 @@
+"""
+empty message
+
+Revision ID: e6f315c1f2f6
+Revises: 92846651ae9d
+Create Date: 2019-10-24 16:26:30.560845
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'e6f315c1f2f6'
+down_revision = '92846651ae9d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('validated_metabolite_table', sa.Column('imputation_info', sqlalchemy_utils.types.json.JSONType(), nullable=True))
```

Comment:
Ideally we should add the imputation info to the existing rows in this database.  What are the implications on the client if this column is null on a table for which imputation was performed?

Reply:
see 
https://github.com/girder/viime/blob/a80832d30b55b63944e6b514d2ca1995cbafdc4f/web/src/components/Impute.vue#L71-L77

I implemented the client to handle null values properly